src/tests/JIT/opt/ValueNumbering/ExceptionSets.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <CLRTestPriority>1</CLRTestPriority>
```

Comment:
is this necessary?

Reply:
Not necessary per se, I just don't think the tests (there will be more) here are valuable enough to be Pri0. The reason being that they're protecting against bugs we have had for a very long time now, and haven't seen anyone hit them.